crates/miden-objects/src/block/proven_block.rs

```diff
@@ -38,18 +39,8 @@ pub struct ProvenBlock {
     /// The header of the block, committing to the current state of the chain.
     header: BlockHeader,
 
-    /// Account updates for the block.
-    updated_accounts: Vec<BlockAccountUpdate>,
-
-    /// Note batches created by the transactions in this block.
-    output_note_batches: Vec<OutputNoteBatch>,
-
-    /// Nullifiers created by the transactions in this block through the consumption of notes.
-    created_nullifiers: Vec<Nullifier>,
-
-    /// The aggregated and flattened transaction headers of all batches in the order in which they
-    /// appeared in the proposed block.
-    transactions: OrderedTransactionHeaders,
+    /// ...
```

Comment:
Are these `/// ...` to be filled in later? If we don't deal with them in this PR, I'd prefer leaving a `TODO` or `TODO(block_refactor)` here.